src/contact-summary/contact-summary-emitter.js

```diff
@@ -81,6 +84,9 @@ function execAppliesIf(prop, report) {
 }
 
 function addCard(card, context, r) {
+  if (typeof card.fields === 'function' && arguments.length < 3) {
+    return;
+  }
```

Comment:
I'm not sure what you're trying to achieve here. Can `fields` only be a function when adding a card about a report? Can we revert this change?

Reply:
Yes. This can be reverted. Yes, that was the impression I got from line 102 about fields and report cards.